### PR TITLE
support external auth mechanism on fluentd sources as well

### PR DESCRIPTION
**Pull request by bcollard**
This option is present on out_amqp but is still missing on in_amqp.
Please patch with v1.13.1?

### CODE REVIEW

File: lib/fluent/plugin/in_amqp.rb [hunk 1]
```diff
@@ -143,6 +144,7 @@ def get_connection_options()
         verify_ssl: @verify_ssl, heartbeat: @heartbeat,
         tls: @tls,
         verify_peer: @tls_verify_peer
+        auth_mechanism: @auth_mechanism
```
[the remark this answers is not in the document]
Comment (bcollard) [reply lib/fluent/plugin/in_amqp.rb]: syntax fixed

**Comment by bcollard** [comment]
FYI, I could build / package it and it works as expected: fluentd connects to rabbitMQ broker with ssl authentication.

**Comment by bcollard** [comment]
any suggestion to merge? @sawanoboly @warmfusion ?

**Comment by warmfusion** [comment]
Looks sensible enough to me.

**Pull request merged**